src/components/Profile/ProfilePage.js

```diff
@@ -0,0 +1,306 @@
+    flexDirection: "row",
+    marginTop: "1rem",
+  },
+  backArrow: {
+    color: "rgba(29,161,242,1.00)",
+  },
+  btn: {
+    border: "1.5px solid rgba(29,161,242,1.00)",
+    color: "rgba(29,161,242,1.00)",
+    fontWeight: "bold",
+    borderRadius: "1.5rem",
+    marginTop: "0.7rem",
+    marginRight: "1rem",
+    textTransform: "capitalize",
+  },
+  lock: {
+    height: "1.5rem",
+    color: "black",
+  },
+  avatarBox: {
+    position: "relative",
+    bottom: "1rem",
+    top: "8rem",
+    marginLeft: "0.7rem",
+  },
+  avatar: {
+    border: "4px solid white",
+    height: "8rem",
+    width: "8rem",
+    marginTop: "7rem",
+  },
+  nameTypo: {
+    color: "black",
+    font: "inherit",
+    fontWeight: "bold",
+    fontSize: "20px",
+  },
+  linksDiv: {
+    display: "flex",
+    flexDirection: "row",
+    marginBottom: "1rem",
+  },
+  links: {
+    textDecoration: "none",
+    font: "inherit",
+    color: "grey",
+    fontWeight: "100px",
+    fontSize: "15px",
+  },
+}));
+
+const ProfilePage = () => {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+  const [tab, setTab] = React.useState("Tweets");
+  const [editProfile, setEditProfile] = React.useState(false);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+  const handleNextTab = () => {
+    let newValue = value;
+    if (newValue !== 3) {
+      newValue = newValue + 1;
+      setValue(newValue);
+    }
+  };
+
+  const handleBackTab = () => {
+    let newValue = value;
+    if (newValue !== 0) {
+      newValue = newValue - 1;
+      setValue(newValue);
+    }
+  };
+
+  const openProfileEditor = () => {
+    setEditProfile(true);
+  };
+
+  return (
+    <Grid container justify="center" spacing={1}>
+      <Grid item xs={12}>
+        <EditProfile
+          open={editProfile}
+          onClose={() => setEditProfile(false)}
+          closeModal={() => setEditProfile(false)}
+        />
+
+        <header className={classes.root}>
+          <div className={classes.div1}>
+            <div>
+              <Button>
+                <KeyboardBackspace className={classes.backArrow} />
+              </Button>
+            </div>
+            <div className={classes.horizontalDiv}>
+              <Typography className={classes.nameTypo} variant="h6">
+                Emi Chukwuka
+              </Typography>
+              <Lock className={classes.lock} />
+            </div>
+          </div>
+        </header>
+      </Grid>
+
+      <main style={{ width: "100%", marginTop: "1rem" }}>
+        <Grid item xs={12}>
+          <Grid item xs={12}>
+            <Paper className={classes.paper}>
+              {" "}
+              <div className={classes.avatarBox}>
+                <Box>
+                  <Avatar className={classes.avatar}>Profile Image</Avatar>
+                </Box>
+              </div>
+            </Paper>
+          </Grid>
+          <Grid style={{ marginLeft: "1rem" }} item xs={12}>
```

Comment:
avoid inline styles. Grid has some inbuilt props you can pass. If that doesn't solve it, move your style to the style object at the top of the file.

Reply:
Okay sir